core/includes/updater.inc

```diff
@@ -149,6 +149,11 @@ abstract class Updater implements BackdropUpdaterInterface {
         unset($info_files[$info_file->uri]);
         continue;
       }
+      // Exclude info files for profiles.
+      if (strpos($info_file->uri, '/core/profiles/') !== FALSE) {
```

Comment:
```suggestion
      if (strpos($info_file->uri, '/core/profiles/') !== FALSE || strpos($info_file->uri, '/profiles/') !== FALSE) {
```

We support contrib/custom profiles under a `/profiles` folder (although we do not ship core with one: https://github.com/backdrop/backdrop-issues/issues/3205), so we should also support that.